### PR TITLE
fix: removed mouse event over in map parcels

### LINKED ISSUE #1750: Prevent mouse over to show land info panel
The behavior of the parcel information panel when you move the mouse over it is often erratic and involuntary, making it not very usable.
Also, independently of how you open del info panel (mouse over or click), if your the pointer selects another parcel in the way, the panel automatically closes. 

**Acceptance criteria**
- [ ] The information panel is only opened by clicking on the parcel.
- [ ] When the panel is open, it only closes if I move the mouse a considerable distance away or click elsewhere (including click and drag)

### PR DESCRIPTION
**Pull request by DavideJensen**
## What does this PR change?

Closes #1750

removes the mouse hold on parcels event in the map
Forces the parcel toast to appear only on the parcel click
Adds the auto closure of the toast when the cursor is at least 40 parcels away
Restores an old explicit test

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=fix/remove-mouse-over-on-map
2. Open the map
3. Test the toast opening and closing with the new behavior

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

**Review by pravusjif — APPROVED** [review]
great job!

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/NavMap/Tests/NavmapToastViewShould.cs [hunk 1]
```diff
@@ -11,14 +11,24 @@ public class NavmapToastViewShould : IntegrationTestSuite_Legacy
...
+        
         [UnitySetUp]
         protected override IEnumerator SetUp()
         {
             yield return base.SetUp();
             yield return null;
 
+            controller = new MinimapHUDController();
             navmapView = Object.FindObjectOfType<NavmapView>();
+            Debug.Log("Navmapview is null? " + navmapView);
```
Comment (BrianAmadori) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/NavMap/Tests/NavmapToastViewShould.cs:31]:
```suggestion
```
Stray debug logs should be removed.

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/NavMap/NavmapView.cs [hunk 2]
```diff
@@ -49,8 +49,7 @@ void Start()
             toastView.OnGotoClicked += () => navmapVisible.Set(false);
 
             MapRenderer.OnParcelClicked += TriggerToast;
-            MapRenderer.OnParcelHold += TriggerToast;
-            MapRenderer.OnParcelHoldCancel += () => { toastView.OnCloseClick(); };
+            MapRenderer.OnCursorFarFromParcel += () => { toastView.OnCloseClick(); };
```
Comment (BrianAmadori) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/NavMap/NavmapView.cs:52]: Beware, I think you forgot to unsubscribe from this method on `OnDestroy()`.

**Review by BrianAmadori — CHANGES_REQUESTED** [review]
Congrats on your first PR! Small

**Review by sandrade-dcl — CHANGES_REQUESTED** [review]
I have tested the map with this new implementation and I'm noticing a weird behaviour: When I click on a parcel, the open animation of the card is not always being executed but it appears already opened instantly. I think this animation should be always executed from the beginning.

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/NavMap/NavmapView.cs (hunk 2 above)
Comment (DavideJensen) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/NavMap/NavmapView.cs]: Done with commit 21990120474e1cd08c47a37a5e7dc5ffd7cc7403

**Comment by DavideJensen** [comment]
> I have tested the map with this new implementation and I'm noticing a weird behaviour: When I click on a parcel, the open animation of the card is not always being executed but it appears already opened instantly. I think this animation should be always executed from the beginning.

Fixed with commit 468d29febed7754648a88033300236d206ce74ec

**Review by sandrade-dcl — APPROVED** [review]
It's working fine now, thanks Davide! Great work!

**Review by BrianAmadori — APPROVED** [review]
LGTM codewise! 🎉

**Pull request merged**